app-connector-client/server/resources/api.yaml

```diff
@@ -260,6 +260,109 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /local/apis:
+    get:
+      tags:
+        - "api"
+      summary: "Gets all local apis bundled from the config file"
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            application/json:
+              schema:
+                type: "array"
+                items:
+                  $ref: "#/components/schemas/Service"
+  /local/apis/registration:

```

Comment:
Maybe we should move the action one level up -> /local/registration
Otherwise the response structure is ambigious, /local/apis/icke returns API, /local/apis/registration returns status

Reply:
removed /apis path from registeration